lib/astarte_core/triggers/policy_protobuf/handler.proto

```diff
@@ -0,0 +1,30 @@
+//
+// This file is part of Astarte.
+//
+// Copyright 2021 Ispirata Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+message Handler {
+    
+    enum StrategyType {
+        DISCARD = 1;
+        RETRY = 2;
+    }    
+    StrategyType strategy = 1;
+    oneof on {
+        KeywordError keyword_error = 2;
```

Comment:
It's feasible, but since `RangeError` can't be unwrapped in the same way (because it contains a `repeated` field), it would break the nice (I dare say *elegant*) symmetry we have